### PR TITLE
Decide if the source should be actually skipped

### PR DESCRIPTION
**Pull request by salty-blue-mango**
### What does this PR do?
Port of #50742

### What issues does this PR fix or reference?
Bugfix. If there is a Debian repo with signed-by option, Salt doesn't show it.

### Tests written?
Yes

### Commits signed with GPG?

No

### CODE REVIEW

File: salt/modules/aptpkg.py [hunk 1]
```diff
@@ -1575,6 +1575,27 @@ def list_repo_pkgs(*args, **kwargs):  # pylint: disable=unused-import
     return ret
 
 
+def _skip_source(source):
```
Comment (dwoz) [inline salt/modules/aptpkg.py]: `_skip_source` should get unit tested

**Review by dwoz — CHANGES_REQUESTED** [review]

File: salt/modules/aptpkg.py (hunk 1 above)
Comment (garethgreenaway) [reply salt/modules/aptpkg.py]: Added a test for _skip_source.

**Review by garethgreenaway — DISMISSED** [review]

**Review by cmcmarrow — DISMISSED** [review]

**Review by frogunder — DISMISSED** [review]

**Comment by dwoz** [comment]
@Nate-Watts Black needs to be run on this

**Comment by dwoz** [comment]
@Nate-Watts Black needs to be run on these changes.

**Review by dwoz — APPROVED** [review]

**Pull request merged**